services/src/services_dynalib.c

```diff
@@ -29,4 +29,5 @@
 #include "system_error.h"
 #include "led_service.h"
 #include "diagnostics.h"
+#include "printf_float.h"
```

Comment:
where does the implementation of `_printf_float` come from?  Do we have any tests to in wiring/no_fixture to validate the behaviour?

Reply:
It's an internal function of newlib. Comes from `lib_a-nano-vfprintf_i.o`. There is a test `STRING_08_sprintf` which `sprintf`'s `%f` and if `_printf_float` is not working the test will fail.